README.rst

```diff
@@ -44,6 +44,13 @@ API's te realiseren die gebruikt kunnen worden bij gemeenten.
 
 .. _`referentie implementaties van VNG Realisatie`: https://github.com/VNG-Realisatie/gemma-zaken
 
+Requirements
+============
+
+* PostgreSQL database
+* Install extensions ``postgres`` and ``uuid-ossp``. If the DB user is a
```

Comment:
Is this still relevant without triggers?

If so, make this a subbullet from the PostgreSQL database bullet. And, shouldn't postgis be here as well (with link to https://postgis.net/install/)?

Reply:
No, it's not relevant now. Fixed.